sys/decl.go

```diff
@@ -236,10 +236,21 @@ func (t FilenameType) Align() uintptr {
 	return 1
 }
 
+type ArrayKind int
+
+const (
+	ArrayRandLen ArrayKind = iota
+	ArrayFixedLen
```

Comment:
Should I just remove the Len field then?
It seems that it's being used by `addAlignment()` in `sys/align.go`.